plug.vim

```diff
@@ -1332,6 +1698,8 @@ function! s:upgrade()
       endif
     elseif s:ruby
       call s:upgrade_using_ruby(new)
+    elseif s:py2
+      call s:upgrade_using_python(new)
     else
       return s:err('curl executable or ruby support not found')
```

Comment:
`'curl executable or ruby support not found'` this string should mention python.